### ISSUE TITLE
microcode-20201110 hangs and does not boot

### ISSUE THREAD

**Issue by cca310** [intel/Intel-Linux-Processor-Microcode-Data-Files#47]
I have a [Lenovo Yoga 710-11ISK](https://pcsupport.lenovo.com/us/en/products/laptops-and-netbooks/yoga-series/yoga-710-11isk/80tx) laptop. The computer has an [intel m3-6Y30](https://ark.intel.com/content/www/us/en/ark/products/88198/intel-core-m3-6y30-processor-4m-cache-up-to-2-20-ghz.html) processor.

I am using ubuntu 21.04. The computer does not boot after [3.20201110.0ubuntu1](https://launchpad.net/ubuntu/+source/intel-microcode/3.20201110.0ubuntu1) microcode update. It says "Loading initial ramdisk".
I am able to boot if I select an older kernel in the grub menu or if I enter ubuntu recovery mode.

 ```
 $ dmesg | grep microcode
 [    0.000000] microcode: microcode updated early to revision 0xd6, date = 2019-10-03
 [    0.154466] SRBDS: Vulnerable: No microcode
 [    1.001093] microcode: sig=0x406e3, pf=0x80, revision=0xd6
 [    1.001217] microcode: Microcode Update Driver: v2.2.
```

There is a similar issue seen here: https://bugs.launchpad.net/ubuntu/+source/intel-microcode/+bug/1904658

I am unsure what logs are useful. Please let me know if there's other information I need to provide

**Comment by crawfxrd** [comment]
## Previous

- coreboot
  - Upstream MR for oryp5 [\[1\]]
  - Determined causes of Jenkins failures
    - Need a new release of intel-ucode for CML-H support
    - But latest ucode release breaks several platforms

## Today

- coreboot
  - Update intel-ucode and add support for building CML-H
    - Rebase oryp6, gaze15 on intel-ucode update so they build in Jenkins
  - Convert system76/oryp5 to clevo/cfl-h
  - Rebase TAS5825M patch on oryp5
- firmware-open
  - Use `defconfig` to generate the coreboot `.config` file

[\[1\]]: https://review.coreboot.org/c/coreboot/+/47892

**Comment by esyr-rh** [comment]
Seems to be the same issue as #31.

**Comment by hxss** [comment]
Same issue on Yoga 710-11ISK with `m5-6Y54`. Last working ucode - `20200616-1`.

**Comment by esyr-rh** [comment]
New revision 0xea of 06-4e-03/06-5e-03 microcode files has been published as part of [microcode-20210608](https://github.com/intel/Intel-Linux-Processor-Microcode-Data-Files/releases/tag/microcode-20210608) release, it may be worth to try it out.

**Comment by cca310** [comment]
The 20210608 release solved my issue
